spec.html

```diff
@@ -210,7 +210,7 @@ <h1>ShadowRealmImportValue ( _specifierString_, _exportNameString_, _callerRealm
 				1. Let _hasOwn_ be ? HasOwnProperty(_exports_, _string_).
 				1. If _hasOwn_ is *false*, throw a *TypeError* exception.
 				1. Let _value_ be ? Get(_exports_, _string_).
-				1. Let _realm_ be _f_.[[ShadowRealm]].
+				1. Let _realm_ be _f_.[[Realm]].
```

Comment:
[[ShadowRealm]] is a Realm record, although the naming is exclusive to ShadowRealms to avoid collision with the existing [[Realm]] existing in ECMA-262.

Reply:
Wait, this is confusing... The object is a ShadowRealm instance, this object has a reference to a [[Realm]], a traditional realm record from 262. Why should we have a `[[ShadowRealm]]` internal slot then?